autoscheduler/scraper/tests/banner_requests_tests.py

```diff
@@ -90,16 +90,32 @@ class BannerRequestsTests(AioTestCase):
 
         # Arrange
         term = "201931" # Fall 2019
-        request = BannerRequests(term)
+        request = BannerRequests()
 
-        depts = ["CSCE", "MATH", "ECEN"]
+        depts = ["CSCE", "ECEN", "MATH"]
+        depts_terms = zip(depts, [term for i in range(len(depts))])
+
+        def spy(course_list, _, __, ___):
```

Comment:
You only have to use 1 underscore to ignore a parameter, also since you're using multiple in a row you can use `*_` instead of repeating